cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp

```diff
@@ -439,13 +439,13 @@ int LuaEngine::handleCommonEvent(void* data)
         return 0;
    
     CommonScriptData* commonInfo = static_cast<CommonScriptData*>(data);
-    if (NULL == commonInfo->eventName || 0 == commonInfo->handler)
+    if ('\0' == commonInfo->eventName[0] || 0 == commonInfo->handler)
```

Comment:
Yep, `eventName` doesn't need to be checked since it's always non-null.
But your change affects the original logic.
The original logic allows to pass in an empty string("") while current logic doesn't allow to pass in an empty string.
Please refer my Pull Request: https://github.com/cocos2d/cocos2d-x/pull/15922/files